src/primitives.jl

```diff
@@ -1,12 +1,22 @@
 """
     Reprint(fn) - apply the lambda function when printed
 """
-mutable struct Reprint
+struct Reprint
```

Comment:
I left `Reprint` in because it is mentioned in the docs, people might be using it?